### ISSUE TITLE
False positive VUID-vkResetCommandBuffer-commandBuffer-00045 error after exporting submitted fence handle

### ISSUE THREAD

**Issue by i-nazarov-ukraine** [KhronosGroup/Vulkan-ValidationLayers#5694]
**Environment:**
 - OS: Android
 - GPU: Any
 - SDK or header version if building from repo: 1.3.247
 - Options enabled: Default

**Describe the Issue**

- Create `VkFence` with `VkExportFenceCreateInfoKHR::handleTypes = VK_EXTERNAL_FENCE_HANDLE_TYPE_SYNC_FD_BIT_KHR`;
- Submit empty `VkCommandBuffer` with created Fence.
- Call `vkGetFenceFdKHR()` on the Fence.
- Wait on the exported FD using `poll()` function.
- Call `vkResetCommandBuffer()` on the Command Buffer.
- The "VUID-vkResetCommandBuffer-commandBuffer-00045" error is generated.

**Expected behavior**

VVL should not generate the error, because it has no means to verify how exported FD was used by the application.
It should behave similarly to `vkQueueSubmit()` when submitting `VkFence` with scope other than `kSyncScopeInternal`.
In other words, submission should be retired early after `vkGetFenceFdKHR()` is called.

**Additional context**

<details>
<summary>terminal output</summary>

```sh
Validation Error: [ VUID-vkResetCommandBuffer-commandBuffer-00045 ] Object 0: handle = 0x6e09023af0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; | MessageID = 0x1e7883ea | Attempt to reset VkCommandBuffer 0x6e09023af0[] which is in use. The Vulkan spec states: commandBuffer must not be in the pending state (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkResetCommandBuffer-commandBuffer-00045)
```

</details>

**Comment by artem-lunarg** [comment]
Fixed by https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/5695
